pyhive/common.py

```diff
@@ -12,11 +12,14 @@
 from past.builtins import basestring
 from pyhive import exc
 import abc
-import collections
```

Comment:
The file still uses deque from collections at https://github.com/dropbox/PyHive/blob/d6e7140d366cc37d01af0d8308e7739d090d73ef/pyhive/common.py#L41. This import should not be removed.

Reply:
reverted to "import collections" and changed import at Iterable statement as collections.abc.Iterable